test/open_mic_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require './lib/open_mic'
+
+class OpenMicTest < Minitest::Test
+  def test_it_exists
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    assert_instance_of OpenMic, open_mic
+  end
+
+  def test_it_has_attributes
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    assert_equal open_mic.location, "Comedy Works"
+    assert_equal open_mic.date, "11-20-18"
+  end
+
+  def test_it_can_welcome_user
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    sal = User.new("Sal")
+    ali = User.new("Ali")
+    open_mic.welcome(sal)
+    open_mic.welcome(ali)
+    assert_equal 2, open_mic.performers.length
```

Comment:
i would specifically check that open_mic.performers is an array with sal and ali inside.  checking that the length of the array is 2 isn't a very useful test-- it could be two strings, two integers, two booleans, two nil values, etc.  Get as specific as you can when making assertions. 

`assert_equal [sal, ali], open_mic.performers`